provision/core/vvv/provision.sh

```diff
@@ -96,8 +96,21 @@ function shyaml_setup() {
   # Used for passing custom parameters to the bash provisioning scripts
   if [ ! -f /usr/local/bin/shyaml ]; then
     vvv_info " * Installing Shyaml for bash provisioning.."
-    sudo pip3 install wheel
-    sudo pip3 install shyaml
+
+    local OSVERSION_NUMBER=$(lsb_release lsb_release -sr)
```

Comment:
shellcheck will probably flag this and a few other things, it should be something like this:
```suggestion
    local OSVERSION_NUMBER
    OSVERSION_NUMBER=$(lsb_release lsb_release -sr)
```

Reply:
adjusted

Reply:
huh, this adjustments causing error for some reason
```
lsb_release: error: No arguments are permitted
```
https://github.com/Varying-Vagrant-Vagrants/VVV/actions/runs/11580749924/job/32239965581?pr=2739

Reply:
reverted it to the original changes. the checks / linting seems to be fine: https://github.com/Varying-Vagrant-Vagrants/VVV/actions/runs/11581054132/job/32240928199?pr=2739
and we already doing these kind of local syntax previously: https://github.com/Varying-Vagrant-Vagrants/VVV/blob/develop/provision/core/vvv/provision.sh#L62-L64

Reply:
ah I think the problem is a mistake on my part, `lsb_release` vs `lsb_release lsb_release`

Reply:
adjusted, and seems to be ran okay: https://github.com/Varying-Vagrant-Vagrants/VVV/actions/runs/11858839047/job/33050382482?pr=2739
let me know if it needs further adjustments